i18n-util/webpack.config.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+module.exports = {
+  mode: "development",
+  entry: './src/main.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'i18n-util.bundle.[fullhash].js'
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
```

Comment:
Can we also at an **auto-generated** notice to this file as a comment?

Reply:
FYI this seems to go away with production mode, so I added a new command for build-dev that runs in development mode with source maps enabled for debugging